### PR TITLE
Refine local kind setup

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What problem does this PR solve? 
Makes local kind script compliant with kind 0.7.0, support specifying helm and kind version through environments

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Build a kind cluster locally and it works fine

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: hack/kind-cluster-build.sh [hunk 1]
```diff
@@ -22,20 +22,32 @@ source $ROOT/hack/lib.sh
...
+want to mutate the defaulting one:
+
+    export KUBECONFIG=${HOME}/.kube/kind-config-<cluster>
 
 Options:
        -h,--help               prints the usage message
        -n,--name               name of the Kubernetes cluster,default value: kind
        -c,--nodeNum            the count of the cluster nodes,default value: 6
        -k,--k8sVersion         version of the Kubernetes cluster,default value: v1.12.8
        -v,--volumeNum          the volumes number of each kubernetes node,default value: 9
```
Comment (cofyc) [inline hack/kind-cluster-build.sh]: should we reduce the default num of the cluster too? 1 control plane and 3 workers are enough for most cases.

File: hack/kind-cluster-build.sh [hunk 2]
```diff
@@ -232,19 +245,19 @@ $HELM_BIN init --service-account=tiller --wait
 # FIXME: remove this
 docker pull gcr.io/google-containers/kube-scheduler:${k8sVersion}
 docker tag gcr.io/google-containers/kube-scheduler:${k8sVersion} mirantis/hypokube:final
-kind load docker-image --name=${clusterName} mirantis/hypokube:final
+$KIND_BIN load docker-image --name=${clusterName} mirantis/hypokube:final
```
Comment (cofyc) [inline hack/kind-cluster-build.sh]: I guess this is not necessary right now. I'm fine to remove this in this PR or another PR.

Comment (aylei) [reply hack/kind-cluster-build.sh]: I'd like to remove it later because it requires additional test

File: hack/kind-cluster-build.sh (hunk 1 above)
Comment (aylei) [reply hack/kind-cluster-build.sh]: good point, changed

**Pull request merged**